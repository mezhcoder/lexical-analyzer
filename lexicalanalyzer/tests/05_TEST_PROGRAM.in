program Test; var x, y: integer; begin x := 2 + 3 * 4; end.