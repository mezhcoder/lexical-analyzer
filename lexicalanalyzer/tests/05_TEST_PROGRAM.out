1	1	KEYWORD	program	program
1	9	ID	Test	Test
1	13	OPERATOR	;	;
1	15	KEYWORD	var	var
1	19	ID	x	x
1	20	OPERATOR	,	,
1	22	ID	y	y
1	23	OPERATOR	:	:
1	25	ID	integer	integer
1	32	OPERATOR	;	;
1	34	KEYWORD	begin	begin
1	40	ID	x	x
1	42	OPERATOR	:=	:=
1	45	INTEGER	2	2
1	47	OPERATOR	+	+
1	49	INTEGER	3	3
1	51	OPERATOR	*	*
1	53	INTEGER	4	4
1	54	OPERATOR	;	;
1	56	KEYWORD	end	end
1	59	OPERATOR	.	.